lib/screens/signup/signup.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:ecellapp/screens/signup/cubit/signup_cubit.dart';
+import 'package:ecellapp/screens/signup/widgets/email_field.dart';
+import 'package:ecellapp/screens/signup/widgets/first_name_field.dart';
+import 'package:ecellapp/screens/signup/widgets/last_name_field.dart';
+import 'package:ecellapp/screens/signup/widgets/password_field.dart';
+import 'package:ecellapp/screens/signup/widgets/mobile_number_field.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class SignupScreen extends StatelessWidget {
+  final _formKey = GlobalKey<FormState>();
+  final TextEditingController firstNameController = TextEditingController();
+  final TextEditingController lastNameController = TextEditingController();
+  final TextEditingController emailController = TextEditingController();
+  final TextEditingController passwordController = TextEditingController();
+  final TextEditingController mobileController = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: BlocConsumer<SignupCubit, SignupState>(
+        listener: (context, state) {
+          if (state is SignupError) {
+            Scaffold.of(context).showSnackBar(
+              SnackBar(
+                content: Text(state.message),
+              ),
+            );
+          }
```

Comment:
Always add TODOs wherever necessary.
```suggestion
          } if (state is SignupSuccess) {
              // TODO add on success logic
          }
```

Reply:
this is also not done yet, and you resolved it anyway...